install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -21,12 +21,51 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
   (NULL, 'actionFrontControllerSetVariables', 'Add variables in JavaScript object and Smarty templates', 'Add variables to javascript object that is available in Front Office. These are also available in smarty templates in modules.your_module_name.', '1'),
   (NULL, 'displayAdminGridTableBefore', 'Display before Grid table', 'This hook adds new blocks before Grid component table.', '1'),
   (NULL, 'displayAdminGridTableAfter', 'Display after Grid table', 'This hook adds new blocks after Grid component table.', '1'),
-  (NULL, 'displayAdminOrderCreateExtraButtons', 'Add buttons on the create order page dropdown', 'Add buttons on the create order page dropdown', '1')
```

Comment:
If I am not foolish these lines were missing the last comma 😮  so it was supposed to fail?

Reply:
@matks https://github.com/PrestaShop/PrestaShop/pull/24242/files#diff-2ab29e497243cd89a7b49b94a35a33d9ca879e40ed9a3cf48853535ec304c423 😅

Reply:
Nice catch, looks like @atomiix was ahead of me one more time 😄 

We have a failing test from autoupgrade which indicates the issue => great! 💪 
https://github.com/PrestaShop/autoupgrade/actions/runs/855151733

But why only the 1.6 -> 1.7 runs have failed 🤔  . Such an obvious SQL issue should break everything?

Reply:
@matks the failing doesn't come from the missing commas but I fixed them during the process of fixing the other issues

Reply:
@atomiix Then I'm scared 😨  if our autoupgrade CLI tests cannot detect a failing SQL statement